app/init/init_gatekeeper.py

```diff
@@ -0,0 +1,46 @@
+import requests
+from core import settings
+
+def register_apis_to_gatekeeper():
+
```

Comment:
Whoops, forgot to check whether the services have already been registered.

Reply:
mmm yes. I was wondering if we need this tbh. In fact every time you re-register endpoints the routes are updated in gatekeeper. So in fact if an endpoint changes a bit this will be updated, otherwise we do not catch it